docs/consensus/security.md

```diff
@@ -13,4 +13,75 @@ keywords:
 - Constraining Simulation
 - Handling Equivocation
 - Detecting Long-Range Attacks -->
-For detailed security analysis please refer to our paper [Dilithium: A Proof-of-Archival-Storage Consensus Protocol for Subspace](https://github.com/subspace/consensus-v2-research-paper)
\ No newline at end of file
+
+A complex system like the Subspace protocol has many potential attack vectors, some are more general, blockchain related, 
+while other are focused on proof of capacity and specifically proof of archive storage, as used in Subspace.  
+
+This page attempts to give an overview of how such attacks are mitigated in the Subspace protocol. There are many things
+ to cover, so the reader should be aware that at the current state this page is not exhaustive. Furthermore, the 
+presentation mostly highlights the techniques used.
```

Comment:
I'm not sure what this sentence is trying to say "Furthermore, the 
presentation mostly highlights the techniques used." It feels incomplete.

Reply:
Trying to say that detailed explanation (furthermore, security proofs) are omitted. I thought the following sentence makes this clear, but I'll revise.